src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -279,42 +261,76 @@
             </div>
           </div>
 
-          <!-- Flags -->
-          <div class="form-group row"
-               *ngIf="info.is_all_bluestore && form.getValue('poolType') === 'erasure'">
-            <label i18n
-                   class="col-form-label col-sm-3">Flags</label>
+          <!-- Crush ruleset selection -->
+          <div class="form-group row">
+            <label class="col-form-label col-sm-3"
+                   for="crushRule"
+                   i18n>Crush ruleset</label>
             <div class="col-sm-9">
-              <div class="custom-control custom-checkbox">
-                <input type="checkbox"
-                       class="custom-control-input"
-                       id="ec-overwrites"
-                       formControlName="ecOverwrites">
-                <label class="custom-control-label"
-                       for="ec-overwrites"
-                       i18n>EC Overwrites</label>
+              <ng-template #noRules>
+                <span class="form-text text-muted">
+                  <span i18n>There are no rules.</span>
+                  <span *ngIf="form.getValue('poolType') === 'erasure'; else noRulesDefault">
+                    <span i18n> A new crush ruleset will be implicitly created.</span>
```

Comment:
I suggest to add the blank before the sentence manually, otherwise this blank is included in the translation text, right?
```suggestion
                    <span>&nbsp;</span><span i18n>A new crush ruleset will be implicitly created.</span>
```

Reply:
and I think it will fail during linting...
Can't we just add the `nbsp `before the `span`? not sure if it needs to be inside a `span`
```suggestion
                    &nbsp;<span i18n>A new crush ruleset will be implicitly created.</span>
```

Reply:
Fixed.